modules/database/src/std/rec/calcRecord.dbd.pod

```diff
@@ -45,7 +45,7 @@ See L<Address
 Specification|https://docs.epics-controls.org/en/latest/guides/EPICS_Process_Database_Concepts.html#address-specification>
 for information on how to specify database links.
 
-=fields INPA, INPB, INPC, INPD, INPE, INPF, INPG, INPH, INPI, INPJ, INPK, INPL
+=fields INPA -INPL
```

Comment:
A bit inconsistent spacing: `INPA -INPL` vs `ONVA - ONVU`

Reply:
And both work 😀

Reply:
Fixed.